src/ui-shell/sidenav/sidenav-item.component.ts

```diff
@@ -31,11 +31,20 @@ import { Router } from "@angular/router";
 
 		<ng-template #sidenavItemRouterTpl>
 			<a
+				[attr.title]="title ? title : null"
 				[routerLink]="route"
+ 				[relativeTo]="routeExtras?.relativeTo"
+				[queryParams]="routeExtras?.queryParams"
+				[fragment]="routeExtras?.fragment"
+				[queryParamsHandling]="routeExtras?.queryParamsHandling"
+				[preserveFragment]="routeExtras?.preserveFragment"
+				[onSameUrlNavigation]="routeExtras?.onSameUrlNavigation"
```

Comment:
`onSameUrlNavigation` is the only one that is not defined by the `routerLink` directive...it seems to be a property only for `router.navigate` which is overriding the default configuration for the router

Reply:
Good catch!